package.json

```diff
@@ -18,7 +18,8 @@
     "memo-parser/{*.js,LICENSE,*.md}"
   ],
   "scripts": {
-    "prebuild": "rimraf lib",
+    "downlevel-get-tsconfig": "sed 's/catch{/catch(_){/g; s/Object.getOwnPropertyDescriptors/Object.getOwnPropertyDescriptors||require(\"object.getownpropertydescriptors\")/g; s/Object.getOwnPropertySymbols/Object.getOwnPropertySymbols||require(\"get-own-property-symbols\")/g' node_modules/get-tsconfig/dist/index.js > src/get-tsconfig.js",
```

Comment:
@ljharb How about this one? We  use the npm module and add compatibility for own supported versions.

Reply:
This is very different with original `vendors` approach.

Reply:
That still requires us to check in minified code output that we didn't author, and then patch it.

Reply:
We have test cases, I don't think that would be a big issue.